assets/queries/k8s/tls_connection_certificate_not_setup/test/negative3.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```

To use `kubelet` commands you need to use kubelet docker image
```suggestion
      image: joaodanielrufino/kubelet
```